src/compiler/transformers/es2015.ts

```diff
@@ -3337,8 +3337,12 @@ namespace ts {
         scoped: false,
         priority: 0,
         text: `
+            var __extendStatics = (this && this.__extendStatics) ||
+                Object.setPrototypeOf ||
+                ({ __proto__: [] } instanceof Array && function (d, b) { d.__proto__ = b; }) ||
+                function (d, b) { for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p]; };

```

Comment:
I think this is our first helper that conditionally includes code. Additionally, if we port this to `tslib`, you have a strange issue which is that even if you create your own `__extendStatics` helper, you can't change the behavior of an already-imported `__extends` helper.

Thoughts @mhegazy & @rbuckton?